.nvmrc

```diff
@@ -1 +1 @@
-12.16.3
+14.21.1
```

Comment:
Why not going to at least `v16`? I personally would do even `v18`

Reply:
This repo historically supports very legacy tooling, so I went with the oldest non-eol LTS. (Personally, I use `n`, so the setting doesn't affect me anyway. I included it for completeness.)